npm install error(windows-Cygwin)-Win7 64

node-v4.4.3

> weex@0.4.0 install:githooks D:\tmp\weex
> bash ./bin/install-hooks.sh

./bin/install-hooks.sh: line 2: $'\r': command not found
./bin/install-hooks.sh: line 7: syntax error near unexpected token `fi'
./bin/install-hooks.sh: line 7:`fi'

npm ERR! Windows_NT 6.1.7601
npm ERR! argv "D:\tmp\nodejs\node.exe" "D:\tmp\nodejs\node_modules\npm\b
in\npm-cli.js" "run" "install:githooks"
npm ERR! node v4.4.3
npm ERR! npm  v2.15.1
npm ERR! code ELIFECYCLE
npm ERR! weex@0.4.0 install:githooks: `bash ./bin/install-hooks.sh`
npm ERR! Exit status 2
npm ERR!
npm ERR! Failed at the weex@0.4.0 install:githooks script 'bash ./bin/install-ho
oks.sh'.
npm ERR! This is most likely a problem with the weex package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     bash ./bin/install-hooks.sh
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs weex
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!
npm ERR!     npm owner ls weex
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     D:\tmp\weex\npm-debug.log

npm ERR! Windows_NT 6.1.7601
npm ERR! argv "D:\tmp\nodejs\node.exe" "D:\tmp\nodejs\node_modules\npm\b
in\npm-cli.js" "install"
npm ERR! node v4.4.3
npm ERR! npm  v2.15.1
npm ERR! code ELIFECYCLE
npm ERR! weex@0.4.0 postinstall: `npm run install:js-framework && npm run instal
l:h5-render && npm run install:githooks && npm run install:components`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the weex@0.4.0 postinstall script 'npm run install:js-framewo
rk && npm run install:h5-render && npm run install:githooks && npm run install:c
omponents'.
npm ERR! This is most likely a problem with the weex package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     npm run install:js-framework && npm run install:h5-render && npm ru
n install:githooks && npm run install:components
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs weex
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!
npm ERR!     npm owner ls weex
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     D:\tmp\weex\npm-debug.log

82 verbose stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:211:5)
83 verbose pkgid weex@0.4.0
84 verbose cwd D:\tmp\weex
85 error Windows_NT 6.1.7601
86 error argv "D:\tmp\nodejs\node.exe" "D:\tmp\nodejs\node_modules\npm\bin\npm-cli.js" "install"
87 error node v4.4.3
88 error npm  v2.15.1
89 error code ELIFECYCLE
90 error weex@0.4.0 postinstall: `npm run install:js-framework && npm run install:h5-render && npm run install:githooks && npm run install:components`
90 error Exit status 1
91 error Failed at the weex@0.4.0 postinstall script 'npm run install:js-framework && npm run install:h5-render && npm run install:githooks && npm run install:components'.
91 error This is most likely a problem with the weex package,
91 error not with npm itself.
91 error Tell the author that this fails on your system:
91 error     npm run install:js-framework && npm run install:h5-render && npm run install:githooks && npm run install:components
91 error You can get information on how to open an issue for this project with:
91 error     npm bugs weex
91 error Or if that isn't available, you can get their info via:
91 error
91 error     npm owner ls weex
91 error There is likely additional logging output above.
92 verbose exit [ 1, true ]

--- a/README.md
+++ b/README.md
@@ -40,7 +40,7 @@ See [SCRIPTS.md](SCRIPTS.md) for more information.
 
 ## For Windows
 
-Please install [Git for Windows](https://git-scm.com/download/win) or [Cygwin](https://www.cygwin.com/).
+Please install [Git for Windows](https://git-scm.com/download/win).
 
 ## FAQ
 

--- a/doc/faq.md
+++ b/doc/faq.md
@@ -2,4 +2,4 @@
 
 ## Command errors in Windows
 
-Please install [Git for Windows](https://git-scm.com/download/win) first, see [For Windows](https://github.com/alibaba/weex/tree/dev#for-windows) for more information.
+Please install [Git for Windows](https://git-scm.com/download/win) and run commands in git-bash.

